src/model/trust.model.ts

```diff
@@ -92,36 +92,38 @@ export type TrustHistoricalBeneficialOwner =
   TrustHistoricalBeneficialOwnerLegal
   | TrustHistoricalBeneficialOwnerIndividual;
 
-interface TrustCorporate {
+export type TrustCorporate = {
+  id?: string
   type: string;
   name: string;
-  date_became_interested_person_day: string;
-  date_became_interested_person_month: string;
-  date_became_interested_person_year: string;
+  date_became_interested_person_day?: string;
+  date_became_interested_person_month?: string;
+  date_became_interested_person_year?: string;
+  ro_address_premises: string;
```

Comment:
Isn's this field meant to be optional?

Reply:
ro_address_premises maps to property name or number on the form. Which on registered office address is not optional on the form. 
I have noticed some extra field that I should make optional based on the web page like county or postcode
https://roe-v13.herokuapp.com/register/trusts/trust-ole